starfish/intensity_table/intensity_table.py

```diff
@@ -416,7 +420,7 @@ def to_expression_matrix(self, regions: Optional[regional.many]=None) -> Express
         """
 
         # create the 2-d counts matrix
-        grouped = self.to_features_dataframe().groupby(['cell_id', 'target'])
+        grouped = self.to_features_dataframe().groupby(['features', 'target'])
```

Comment:
string constants alert.  also, it's pretty likely this wouldn't have broken if we had used the enums. :)